sqlalchemy_redshift/commands.py

```diff
@@ -310,10 +300,20 @@ def visit_unload_from_select(element, compiler, **kw):
     """
     el = element
 
+    if el.format == Format.csv:
+        format_ = 'FORMAT AS CSV'
+        if el.delimiter is not None or el.fixed_width is not None:
+            raise ValueError(
+                'CSV format cannot be used with delimiter or fixed_width')
+    elif el.format is None:
+        format_ = ''
+    else:
+        raise ValueError('Only CSV format is currently supported')
```

Comment:
Is this a substantially better experience for users if this error is raised at the dialect level rather than letting the query pass to Redshift and get rejected there?

If it doesn't much matter, I'd be inclined to remove this check so that we automatically are compatible with future formats that are already in our enum.

Reply:
Hmm.. I added this because the copy command will raise an exception for formats it doesn't recognize, via check_enum.  It seems a bit inconsistent to have the format validated to the right set of valid formats for copy but not unload, but I don't really have a strong opinion here.

Another option would be to have separate `UnloadFormat` and `CopyFormat` enums, though renaming `Format` to `CopyFormat` would be a breaking change, so that's probably not good.

I'll go ahead and add a commit to remove this exception in a little while.

Reply:
>  It seems a bit inconsistent to have the format validated to the right set of valid formats for copy but not unload, but I don't really have a strong opinion here.

That's a good point. Consistency with existing commands is probably more important than the chance of getting forwards-compatibility. Let's leave this as-is.

Reply:
haha ok.  i'll undo c87aed6